### PR TITLE
Create pull_request_template.md

### LINKED ISSUE #74: Adding template for issues and pull requests
To achieve a higher quality in the issues and in the pull requests we might consider adding templates for both, the issues and the pull requests. The templates are quite easy to create, but what should go in there?
- issues: What kind of issue is committed and what is the relevant information that should be provided by the issue.  It could also include a hint to stick to the code of conduct. 
-  pull requests: Adding a checklist, if everything is complete and all required information is contained.

### PR DESCRIPTION
**Pull request by anilturkmayali**
A new template for pull requests is added that groups the content in three basic sets and also has a link to the Contributing.md.

Closes #74

### CODE REVIEW

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+
+:zap: Please ensure that you reviewed the [guidelines for contributing](/CONTRIBUTING.md) to this repository. 
+
+
+What does this pull request contain? Please explain your changes.
```
Comment (ssteinbuss) [inline docs/pull_request_template.md:6]: We should consider maintaining a changelog (a CHANGELOG.md) and feed directly into it. Then we could just link there and make here a check if the changelog was fed. An example would be the [CHANGELOG.md of the DataSpace Connector](https://github.com/International-Data-Spaces-Association/DataspaceConnector/blob/main/CHANGELOG.md). This needs to go to our [Contributing.md](https://github.com/International-Data-Spaces-Association/idsa/blob/main/CONTRIBUTING.md) like in the [DSC](https://github.com/International-Data-Spaces-Association/DataspaceConnector/blob/main/CONTRIBUTING.md#changelog), they refer to [keepachangelog](http://keepachangelog.com/).

https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

File: docs/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+---------------------------------------------------
+* ...
+
+* ...
+
+* ...
+
+Does this close any currently open issue(s)?
+------------------------------------------
+…
```
Comment (ssteinbuss) [inline docs/pull_request_template.md:16]: Please use the GitHub tooling for this by making use of the [right keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue),

File: docs/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+
+Any other comments?
+-------------------
+…
```
Comment (ssteinbuss) [inline docs/pull_request_template.md:21]: Does this create new issues or are any other issues or pull requests affected by this?

**Review by ssteinbuss — CHANGES_REQUESTED** [review]
Please consider the aspects mentioned:
- keywords for closing issues
- adding new issues
- keep a changelog

**Review by anilturkmayali — COMMENTED** [review]
Thank you for the feedback! Changes applied.

**Pull request merged**